docs/articles/expensify-classic/expenses/Create-Expense-Rules.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Create Expense Rules
+description: Automatically categorize, tag, and report expenses based on the merchant's name
+
+---
+Expense rules allow you to automatically categorize, tag, and report expenses based on the merchant’s name.
+
+# Create expense rules 
+
+1. Hover over **Settings** and click **Account**. 
+2. Click **Expense Rules**.
+2. Click **New Rule**.
+3. Add what the merchant name should contain in order for the rule to be applied. *Note: If you enter just a period, the rule will apply to all expenses regardless of the merchant name. Universal Rules will always take precedence over all other expense rules.*
+4. Choose from the following rules:
+- **Merchant:** Updates the merchant name (e.g., “Starbucks #238” could be changed to “Starbucks”)
+- **Category:** Applies a workspace category to the expense
+- **Tag:** Applies a tag to the expense (e.g., a Department or Location)
+- **Description:** Adds a description to the description field on the expense
+- **Reimbursability:** Determines whether the expense will be marked as reimbursable or non-reimbursable
+- **Billable**: Determines whether the expense is billable
+- **Add to a report named:** Adds the expense to a report with the name you type into the field. If no report with that name exists, a new report will be created if the "Create report if necessary" checkbox is selected. 
+
+![Insert alt text for accessibility here](https://help.expensify.com/assets/images/ExpensifyHelp_ExpenseRules_01.png){:width="100%"}
```

Comment:
Need to actually include the alt text here!

Reply:
That was like that in the original and I didn't even catch it 🤦 I'll get this updated!

Reply:
I updated all 3 images